docs/requirements.txt

```diff
@@ -2,3 +2,8 @@ pyenchant
 Sphinx>=4.5.0
 sphinxcontrib-spelling
 blacken-docs
+
+# Due to the viewcode extension, the docs are dependent on django itself.
+# Below we install django from the local directory. This assumes installation
+# from the project root i.e. `python -m pip install -r docs/requirements.txt`
```

Comment:
Oh, maybe this is why the recommendation before is to execute this from the top level. Perhaps let's add a clarification in the previous paragraphs, `It's necessary to run this command from the project top level folder because...`

Reply:
Made some updates :+1: